### ISSUE TITLE
[Feature Request] Making mouse scroll, scrolling the page up and down instead of changing page.

### ISSUE THREAD

**Issue by lordville951** [tachiyomiorg/tachiyomi#2515]
I am reading Manga horizontally/landscape on my phone so that I get the biggest viewing area. I then scroll down each page from top to bottom reading all the segments, then I switch to the next page.

When I use a Bluetooth controller or Mouse, scrolling doesn't actually scroll the page in Tachiyomi, it just go to the next page. I tried every kind of button on a keyboard as well, and nothing seem to be able to scroll the page.(was trying to find a way to bypass it with keybinds)

It would help me a lot if the scroll function actually scrolls the page up and down Instead of switching page. I can make this kinda work with the Webtoon viewer already, but it scrolls in increments and is hard to follow.

Best regards.

**Comment by arkon** [comment]
Similar request for Fire TV/other D-pad interfaces: #2220

**Comment by arkon** [comment]
Scrolling in an emulator scrolls the page up/down at least. I don't actually have a Bluetooth mouse or keyboard on hand to actually try it on my phone.

**Comment by arkon** [comment]
Just confirmed that a mouse actually does scroll the pages.

**Comment by giulianoriccio** [comment]
I'm using a DS4.
The d-pad/l-analog act the same way and scroll the page for about the 75% of the device height (based on current screen rotation).

When used in chrome the d-pad scrolls for much less and the l-analog triggers an autoscroll with speed based on the analog angle.

It would be nice to emulate the same behavior or in alternative to customize the percentage of the scroll.

**Comment by arkon** [comment]
Closing in favour of #2161
